src/audio/base_fw_intel.c

```diff
@@ -129,10 +129,12 @@ static int basefw_mem_state_info(uint32_t *data_offset, char *data)
 	tuple_data[index++] = info.free_phys_mem_pages;
 	tuple_data[index++] = info.ebb_state_dword_count;
 	for (i = 0; i < info.ebb_state_dword_count; i++) {
-		tuple_data[index + i] = io_reg_read(SHIM_HSPGCTL(i));
+#ifdef INTEL_ADSP
+		tuple_data[index++] = HPSRAM_REGS(i)->HSxPGCTL;
+#else
```

Comment:
@kv2019i do we still need the ifdef now ? since this file is Intel only ?

Reply:
Ack @lgirdwood , this can be dropped now.